test/dnf5daemon-server/support.py

```diff
@@ -70,10 +70,13 @@ def setUp(self):
         self.iface_session = dbus.Interface(
             self.bus.get_object(DNFDAEMON_BUS_NAME, DNFDAEMON_OBJECT_PATH),
             dbus_interface=IFACE_SESSION_MANAGER)
+        # Prevent loading plugins from host by redirecting of pluginconfpath and pluginpath to an empty directory
         self.session = self.iface_session.open_session({
             "config": {
                 "config_file_path": self.config_file_path,
                 "installroot": self.installroot,
+                "pluginconfpath": os.path.join(self.installroot, "pluginconfpath"),
```

Comment:
If the directory is supposed to be empty, I would call it "empty" instead of "pluginconfpath" or "pluginpath".
Also if you do not create the directory and thus the directory does not exist, I would simply use "/dev/null" instead of an artificial path.

Reply:
Unless you want to address this, feel free to merge your current, 95cf7d0121b98e3743867e9e943e2ab909a54716, pull request.

Reply:
I will modify it and thank you very much for the suggestion.